tests/test_csc.py

```diff
@@ -815,6 +817,91 @@ async def test_show_alarms(self):
                 assert data.acknowledgedBy == ""
             assert seen_alarm_names == alarm_names
 
+    @patch("aiohttp.ClientSession.post")
+    async def test_make_log_entry(self, mock_post):
+        """Test the makeLogEntry command."""
+
+        def dict_subset(smallDict, largeDict):
```

Comment:
method parameters should be `snake_case`...

However, do you really need a method just for that? seems like unnecessary overhead for something you can do inline.

Reply:
I think the overhead is insignificant compared to issues of understandability, readability, and reuse. I've used this technique before and expect I will again.  Maybe someone will use it in this file.